src/VisualStudio/CSharp/Impl/LanguageService/CSharpLanguageService_ICSharpProjectHost.cs

```diff
@@ -16,6 +17,12 @@ internal partial class CSharpLanguageService : ICSharpProjectHost
         public void BindToProject(ICSharpProjectRoot projectRoot, IVsHierarchy hierarchy)
         {
             var projectName = Path.GetFileName(projectRoot.GetFullProjectName()); // GetFullProjectName returns the path to the project file w/o the extension?
```

Comment:
We should do that in master - these types aren't thread affinitized right now (though they should be).